Makefile

```diff
@@ -63,7 +63,7 @@ distclean: clean
 	make distclean -C test/auth
 	rm -rf $(RUST_SERVER_API)
 	rm -rf $(RUST_CLIENT_API)
-	rm openapitools.json
+	# rm openapitools.json
```

Comment:
I think it's better to delete the line. Unless commented out is still useful - a comment would be welcome.

Reply:
Done